src/Components/Card.tsx

```diff
@@ -114,23 +87,34 @@ export const Card = ({
   costValue,
   costQualifier,
   age,
-  categories,
 }: ServicePreview): JSX.Element => {
-  const theCats = [
-    categories?.category1 ? categories?.category1 : '',
-    categories?.category2 ? categories?.category2 : '',
-  ].filter(Boolean)
+  const ageDisplay = age ? formatAgeDisplay(age.minAge, age.maxAge) : null
 
-  const theAge = age ? formatAgeDisplay(age.minAge, age.maxAge) : null
-
-  const theCost =
+  const costDisplay =
     costValue || costValue === 0 || costQualifier
       ? formatCostDisplay(costValue, costQualifier)
       : null
 
+  const cardLink = useRef<HTMLAnchorElement | null>(null)
+
   return (
-    <CardContainer>
+    <CardContainer onClick={() => cardLink.current?.click()}>
+      <Description>
+        <Link href={`/service/${id}`} passHref>
+          <a ref={cardLink}>{shortDescription}</a>
```

Comment:
Why bring a `ref` into this, rather than just wrapping the entire card in this link? Is this an accessibility thing?

Reply:
Yep - we could just wrap with a link but I think the link text would be quite verbose if we include the whole card content.

Reply:
Links are also not meant to contain other interactive content, and we'll want the card to have a button to save to favourites.